config/resources/test_config_null_nested_fields.json

```diff
@@ -67,6 +67,7 @@
   },
   "proxy": {
     "config": null,
+    "publicListenerPort": 0,
```

Comment:
This is a `null`able type according to the schema. Do we want to test `null` here instead of `0`?